runtime/jcl/module.xml

```diff
@@ -211,6 +226,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<library name="j9thr"/>
 			<library name="socket" type="macro"/>
 			<library name="sunvmi"/>
+			<library name="criu" type="external">
+				<include-if condition="spec.flags.opt_criuSupport"/>
+				<include-if condition="spec.linux.*"/>
```

Comment:
The library will be included if any `<include-if>` condition is satisfied, which I don't think is what you mean.

Reply:
Is the criu library available on any non-Linux platform? If not, I think the fix should have been
```
    <include-if condition="spec.flags.opt_criuSupport and spec.linux.*"/>
```